app.js

```diff
@@ -0,0 +1,58 @@
+// var ip = document.querySelector("#ip");
```

Comment:
Remove unnecessary code, though it is commented still this is not a good practice to have